src/bin/new.rs

```diff
@@ -45,11 +47,19 @@ pub fn execute(options: Options, config: &Config) -> CliResult<Option<()>> {
                           options.flag_frozen,
                           options.flag_locked));
 
-    let Options { flag_bin, arg_path, flag_name, flag_vcs, .. } = options;
+    let Options { flag_bin, flag_lib, arg_path, flag_name, flag_vcs, .. } = options;
+
+    let is_bin = if !flag_lib {
+        true
+    }
+    else {
+        flag_bin
+    };
```

Comment:
Oh I see, yeah let's just pass through these flags as-is and have the "default logic" kick in during the implementation below